### ISSUE TITLE
timeGroup macro returns: Could not apply macros: error parsing interval $__interval

### ISSUE THREAD

**Issue by chrisleavoy** [grafana/google-bigquery-datasource#106]
Greetings, doc are fairly light on how do we use the $__timeGroup() macro... This query used to work:

```sql
SELECT
  $__timeGroup(timestamp,$__interval),
  _TABLE_SUFFIX as metric,
  count(1) as events
FROM `$dataset.*`
WHERE $__timeFilter(timestamp)
GROUP BY 1,2
ORDER BY time,metric
LIMIT 150000
```

But now returns:

`Could not apply macros: error parsing interval $__interval`

**Comment by dprokop** [comment]
@oscarkilhed do you mind having a look into this?

**Comment by dprokop** [comment]
@chrisleavoy I think the query works, it's just the validation that produces a false result. This will soon be fixed and released with #109 merged.
